deploy/charts/emqx/templates/configmap.acl.yaml

```diff
@@ -0,0 +1,15 @@
+{{ if .Values.emqxAclConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "emqx.fullname" . }}-acl
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: {{ include "emqx.name" . }}
+    helm.sh/chart: {{ include "emqx.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+data:
+  "acl.conf": |
+    {{ .Values.emqxAclConfig }}
+{{ end }}
```

Comment:
We have a style linter which checks:
all the text files in this repo needs a “\n” at the last line

Reply:
Addressed in latest commit